testutil/daemon/daemon.go

```diff
@@ -810,15 +809,6 @@ func (d *Daemon) Info(t testing.TB) types.Info {
 	return info
 }
 
-// cleanupMount unmounts the daemon root directory, or logs a message if
-// unmounting failed.
-func cleanupMount(t testing.TB, d *Daemon) {
-	t.Helper()
-	if err := mount.Unmount(d.Root); err != nil {
```

Comment:
ow come it worked on Windows before?

Reply:
It happened not to be used in tests that are running on Windows 😞 

Likewise, I recall noticing in https://github.com/containerd/cri/pull/1416 that `RecursiveUnmount` "worked" on Windows, because this slice would always be empty https://github.com/moby/moby/blob/v19.03.13/pkg/mount/mount.go#L142